### PR TITLE
Revamp settings pages

### PR DESCRIPTION
**Pull request by tobias-mintlify**

### CODE REVIEW

File: docs/components/accordion.mdx [hunk 1]
```diff
@@ -39,6 +39,8 @@ description: "Using the Accordion component"
 
 The `AccordionGroup` component can be used to group Accordions into a single display.
 
+`AccordionGroup` does not have any props.
```
Comment (hanywang2) [inline docs/components/accordion.mdx:42]: Wrap with a tip component or something

File: docs/settings/customization.mdx [hunk 2]
```diff
@@ -37,12 +31,12 @@ Path to logo image or object with path to "light" and "dark" mode logo images
   Path to the logo in dark mode
 </ResponseField>
 
-</Expandable>
-
+<ResponseField name="href" type="string" default="/">
+  Where clicking on the logo links you to
 </ResponseField>
```
Comment (hanywang2) [inline docs/settings/customization.mdx:36]: We should probably implement this alongside the docs changes

Comment (hahnbeelee) [reply docs/settings/customization.mdx:36]: agreed

File: docs/settings/page.mdx [hunk 3]
```diff
@@ -1,66 +1,73 @@
...
+
+You can show a different title in the navigation with the `sidebarTitle` metadata. This is useful if your title is long and you want something shorter in the navigation links.
+
+## Search Engine Optimization
+
+You can set meta tags like the image set when shared on social media by passing them into your page's metadata. Meta tags with colons need to be wrapped in quotes.
+
+```md Example
+"twitter:image": "/images/your-photo.jpg"
+```
```
Comment (hahnbeelee) [inline docs/settings/page.mdx:73]: amazing

**Review by hahnbeelee — APPROVED** [review]
lgtm! thanks for doing this

**Pull request merged**